internal/concurrency/concurrency.go

```diff
@@ -1,16 +1,16 @@
 package concurrency
 
-// SplitTestFiles splits test files into groups to match concurrency
-func SplitTestFiles(files []string, concurrency int) [][]string {
+// SplitTests splits tests into groups to match concurrency
+func SplitTests(items []string, concurrency int) [][]string {
```

Comment:
I think we should call this method `BinPack`, named after the [problem](https://en.wikipedia.org/wiki/Bin_packing_problem) it's trying to solve.

Likewise, we should update the docs here as well, since the method does not care what it's trying to bin pack, so it shouldn't talk about _test files_ or _tests_.